utils/tika_client.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+
+class TikaClient:
```

Comment:
Dodaj tu jeszcze wykrycie języka może od razu. Bo chcielibyśmy to mieć w atrybutach korpusu